docs/markdown/authoritative/backend-generic-mysql.md

```diff
@@ -0,0 +1,87 @@
+# Generic MySQL backend
```

Comment:
Someone could still create a MySQL backend not based on gsql (fat chance, but still)